### ISSUE TITLE
Statistics shows your score as the number of points, not the percent

### ISSUE THREAD

**Issue by jadebuckwalter** [Aspine/aspine#172]
More specifically, the purple bar that shows your score when you click on the stats for an assignment shows the number of points you got rather than the percentage out of the total points. So, if an assignment is out of 20 points, even if you got a 20/20, the purple bar is at 20 rather than 100.

**Comment by psvenk** [comment]
I think this depends on the teacher, so we should detect when teachers give statistics in percentages and when they give them in points, because (from my experience) most teachers give statistics in percentages (though I think I know to which teacher you are referring).

**Comment by jadebuckwalter** [comment]
I only have one teacher this semester that provides statistics, so this one class is my reference point. However, looking at past years on Aspen, I'm pretty sure that all statistics are shown as point values rather than percentages, unless of course the assignment is out of 100 in which case they would be the same.

**Comment by psvenk** [comment]
Sorry, I mistyped. I meant that most teachers in my experience give statistics in *points* and this one teacher is the only one who gives statistics in percentages.

**Comment by psvenk** [comment]
On hold for now because the statistics-fetching infrastructure will be reworked as part of #184. Let's revisit this issue once that is done.
